src/initializers/kafka/kafka.ts

```diff
@@ -160,3 +170,15 @@ function getAuthOptions(options: {
   const { username, password } = options.sasl || {};
   if (username && password) return { sasl: options.sasl, ssl: options.ssl };
 }
+
+export const isHealthy = async () => {
+  if (!heartBeatConsumer) return false;
+  if (Date.now() - lastHeartbeat < kafkaDefaultSessionTimeout) return true;
+
+  try {
+    const temp = await heartBeatConsumer.describeGroup();
```

Comment:
❓ Is this a network request? I am worried we will make health endpoint slow.

Reply:
It was yes, but it would be used only during rebalancing. Now that we changed to check only producer it isn't needed